_projects/Lossy_checkpoint_restart.md

```diff
@@ -34,19 +34,26 @@ In contrary to previous research that concentrated on few applications, we focus
 
 From our observations of restart from lossy checkpoints in dynamic simulations, we have observed that diffusion and advection problems are reacting differently. We are currently characterizing and formalizing this difference. We suspect that lossyness may be increased as the execution progresses: restarting from a lossy checkpoint near the beginning of the execution seems to affect much more the end results compared with restarting from a lossy checkpoint near the end of the execution. Moreover, we note that in a given variable field, some regions of the field are more chaotic than others. Some applications may not tolerate lossy compression on the entire field. But we suspect that they may tolerate lossy compression in the nonchaotic regions of the variable field. 
 
+## Results for 2016/2017
+
+We particularly explored an important problem: how to bound the impact of  restarting from lossy checkpoint and guarantee that this impact does not affect the application result quality. To address this problem, we established a link between the compression error and the numerical error of the application. Applications using numerical methods suffer errors from truncation and discretization. We showed that compressing checkpoint with an error lower than the numerical errors allow to reserve the quality of the application results. We also demonstrated empirically that the error introduced by restarting from lossy checkpoint can be bounded.    
+
+Other researchers will be able to exploit the link we established between the compression error and the numerical errors to design better compression algorithms and numerical methods that tolerate better compression errors.
+
+An important impact of this work on other disciplines that are using numerical simulation is that they can use lossy compression for checkpoint/restart since we established and verified guidance to fix the compression error that guarantee the quality of the numerical results.
+
 ## Visits and meetings
 
 Franck Cappello visits UIUC almost every week. We have a 30 minutes to 1 hour meeting almost each time.
+Jon Calhoun did an internship of 11 weeks at ANL.
 
 ## Impact and publications
 None yet.
-{% bibliography --cited --file jlesc.bib %}
-
+\cite{Calhoun17}
 
```

Comment:
Also, see e.g. https://github.com/JLESC/jlesc.github.io/blob/source/_projects/exec_interfer_project.md for an example.